hi-ml/src/health_ml/experiment_config.py

```diff
@@ -9,7 +9,6 @@ class ExperimentConfig(param.Parameterized):
                                                   "job in AzureML.")
```

Comment:
It would be helpful to update the description for the param `cluster` to say 'to run on your local machine, omit this argument' or something along those lines